packages/camera/camera_web/example/integration_test/camera_test.dart

```diff
@@ -297,5 +298,175 @@ void main() {
         expect(camera.videoElement.srcObject, isNull);
       });
     });
+
+    group('startVideoRecording', () {
+      testWidgets('starts a video recording', (tester) async {
+        final camera = Camera(
+          textureId: 1,
+          cameraSettings: cameraSettings,
+        );
+
+        await camera.initialize();
+        await camera.play();
+
+        await camera.startVideoRecording();
+
+        expect('recording', camera.mediaRecorder!.state);
+      });
+
+      testWidgets(
+          'starts a video recording with a given maxDuration '
+          'emits a VideoRecordedEvent', (tester) async {
+        // TODO: MediaRecorder with a Timeslice does not seem to call the dataavailable Listener in a Test
```

Comment:
I could not get this test to work. As it does not seem to trigger the datavailable event of the media recorder.
I was able to manually test the behaviour with setting a max duration (I tested with 4 seconds) however this test seems to timeout waiting for the first element of the event stream.
Could you take a look at what's might be going on? 
Also is there a (nice) way to debug the integration tests?

Reply:
I will have a look at this test.

There is a way to debug the integration tests in the browser. You'll need to execute this in `camera_web/example`:

```
flutter run -d web-server --target integration_test/camera_test.dart --debug
```

It will produce a link that you should open in the browser, open inspection tools (CMD+Option+J in Chrome), find a dart file you want to debug in the "Source" tab and mark a breakpoint. Refresh the web page and it should stop at the breakpoint.